scripts/dev/typescript/params.sh

```diff
@@ -150,6 +150,11 @@ function extractParams() {
                 launchFrontend=true
```

Comment:
`envType=dev` ?

Reply:
Yeah..  no other point in lunching unless in dev.. if there is you should explicitly set env